omniscidb/QueryEngine/ColumnFetcher.cpp

```diff
@@ -239,6 +240,11 @@ const int8_t* ColumnFetcher::getAllTableColumnFragments(
   int db_id = col_info->db_id;
   int table_id = col_info->table_id;
   int col_id = col_info->column_id;
+  if (col_info->type->isString() || col_info->type->isArray()) {
```

Comment:
This should probably be a CHECK instead. It is an internal error, not some input error useful for the user.